src/app/portlets/dot-edit-page/content/components/dot-form-selector/dot-form-selector.component.spec.ts

```diff
@@ -0,0 +1,178 @@
+/* tslint:disable:no-unused-variable */
+import { DebugElement, Input, Component, Injectable } from '@angular/core';
+import { DotFormSelectorComponent } from './dot-form-selector.component';
+import { ComponentFixture, TestBed, async, tick, fakeAsync } from '@angular/core/testing';
+import { DOTTestBed } from '../../../../../test/dot-test-bed';
+import { By } from '@angular/platform-browser';
+import { PaginatorService } from '../../../../../api/services/paginator';
+import { Observable } from 'rxjs/Observable';
+import { MockDotMessageService } from '../../../../../test/dot-message-service.mock';
+import { DotMessageService } from '../../../../../api/services/dot-messages-service';
+import { MessageKeyDirective } from '../../../../../view/directives/message-keys/message-keys.directive';
+import { DotDialogModule } from '../../../../../view/components/dot-dialog/dot-dialog.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+const mockContentType = {
+    clazz: 'com.dotcms.contenttype.model.type.ImmutableWidgetContentType',
+    defaultType: false,
+    fixed: false,
+    folder: 'SYSTEM_FOLDER',
+    host: null,
+    name: 'Hello World',
+    owner: '123',
+    system: false
+};
+
+@Injectable()
+class PaginatorServiceMock {
+    url = '';
+
+}
+
+const messageServiceMock = new MockDotMessageService({
+    'contenttypes.form.name': 'Name',
+    'Select': 'Select',
+    'modes.Add-Form': 'Add Form'
+});
+
+xdescribe('DotFormSelectorComponent', () => {
```

Comment:
???

Reply:
Test are broken for a weird reason that non Freddy R or me were able to identify, I don't want to waste time on this right now.